pyproject.toml

```diff
@@ -39,7 +39,6 @@ pyarrow = ">=4.0.0"
 grpcio-status = ">=1.56.0"
 faiss-cpu = "^1.7.4"
 sentence-transformers = "^2.2.2"
-flagembedding = "^1.1.1"
 babel = "^2.12.1"
```

Comment:
Can we remove babel & torch?

Reply:
we need babel for this line in `wiki_sql.py`:
```
from babel.numbers import parse_decimal, NumberFormatError
```

and we need to specify torch version to avoid this error: https://stackoverflow.com/questions/76327419/valueerror-libcublas-so-0-9-not-found-in-the-system-path

Reply:
> we need babel for this line in wiki_sql.py

wiki_sql is just for benchmark. We can have a README about how to run it. Let's remove babel and keep torch then